lib/text/ui_text_displayer.js

```diff
@@ -269,7 +278,9 @@ shaka.text.UITextDisplayer = class {
     }
     if (updateDOM) {
       for (const cueElement of toUproot) {
-        container.removeChild(cueElement);
+        if (cueElement.parentElement) {
+          cueElement.parentElement.removeChild(cueElement);
```

Comment:
I wonder if it would make sense, instead of using an if-statement, to assert that the element has a parent. If it was put into the `toUproot` array it should, right?

Reply:
No, it could have been removed from its parent already.  In particular, in the case of region elements, which are shared among many cues and could be in toUproot multiple times.

Reply:
Added a comment.